File: src/main/java/vocadb/notification/reader/client/model/Language.java
package vocadb.notification.reader.client.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum Language {
    @JsonProperty("Unspecified")
    UNSPECIFIED,
    @JsonProperty("Japanese")
    JAPANESE,
    @JsonProperty("Romaji")
    ROMAJI,
    @JsonProperty("English")
    ENGLISH
}

File: src/main/java/vocadb/notification/reader/client/model/enums/Language.kt
package vocadb.notification.reader.client.model.enums

import com.fasterxml.jackson.annotation.JsonValue

enum class Language(@JsonValue val value: String) {
    UNSPECIFIED("Unspecified"),
    JAPANESE("Japanese"),
    ROMAJI("Romaji"),
    ENGLISH("English");
}
